src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -220,13 +220,13 @@ public ReactionPaginationAction retrieveReactionUsers(@Nonnull String unicode)
     }
 
     @Override
-    public MessageReaction.ReactionEmote getReactionByName(@Nonnull String name)
+    public MessageReaction.ReactionEmote getReactionByUnicode(@Nonnull String unicode)
     {
-        Checks.notNull(name, "Reaction name");
+        Checks.notNull(unicode, "Reaction name");
```

Comment:
Should not this better use `Checks.noWhitespace(unicode, "Reaction unicode");` instead as it checks for `notNull` too?

Reply:
Yeah good catch, thanks

Reply:
Good catch, thank you